portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,67 @@
+html, body {
+  height: 100%;
```

Comment:
Put declarations in alphabetical order in order to achieve consistent code in a way that is easy to remember and maintain: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order